pkg/port/iscsit/session.go

```diff
@@ -286,6 +286,22 @@ func (tq *taskQueue) Pop() interface{} {
 	return item
 }
 
+func (tq *taskQueue) Delete(x interface{}) {
+	item := x.(*iscsiTask)
+	old := *tq
+	n := len(old)
+	for i, t := range old {
+		if t == item {
+			if i == 0 && n == 1 {
+				tq = nil
+				break
+			}
+			*tq = append(old[:i], old[i+1:]...)
+			break
+		}
+	}
```

Comment:
Is it all OK if the item is not found and hence not deleted. Any logs to be printed.?

Reply:
this function is only entered when there is an item to be deleted, added log will be an unreachable code